src/components/ObjektDisplay.vue

```diff
@@ -0,0 +1,48 @@
+<script setup lang="ts">
+    const objektsQuery = `
+        query MyQuery 
+            {collections(limit: 18){
+                front
+                id
+            }
+        }
+        `;
+        
+    const response = await fetch("https://cosmo-api.gillespie.eu/graphql", {
+        method: "POST",
+        headers: {
+            "Content-Type": "application/json"
+        },
+        body: JSON.stringify({
+            query: objektsQuery,
+        })
+    });
+
+    const parsedData = await response.json();
+    const imageList = parsedData.data.collections
+
+
+
+    console.log(imageList);
+</script>
+
+<template>
+    <div id="title">
+        <h2>10 first objekts query</h2>
+    </div>
+    
+    <div class="container">
+        <img v-for="singleImage in imageList" v-bind:src="singleImage.front" v-bind:alt="singleImage.id" width="100%"/>
+    </div>
+</template>
+
+<style>
+/* div.container img{
+    border: 1px dotted aqua;
+} */ 
+.container{
+    display: grid;
+    grid-template-columns: 18% 18% 18% 18% 18%;
+    gap: 2%;
```

Comment:
Now you only filled up 98%
5 * 18%
4 * 2% (since gap is only in between)

You can do the following instead:

```css
  grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
  gap: 1rem;
```

1fr is one fraction. So it is equally distributing them now as you wanted. However, now you can define the gap however you want (like in rem) and it will just adjust itself. No left-over space.

You can also use the repeat function to not type out 1fr 5 times, but rather say "5 times 1fr":

```css
  grid-template-columns: repeat(5, 1fr);
  gap: 1rem;
```

Choosing to fetch 18 objekts is a bit odd, though, because now it won't fit exactly with 5 long rows.

Reply:
Okay, will make the changes. 

I wasn't aware of the fr or rem. saw px or % only. didn't wanted to overfill the page so I kept it 

Requested 18 to check how the grid would react, forgot to change it back.